README.md

```diff
@@ -324,6 +324,7 @@ not to pass them through the command line or environment variables:
 | no-auto-timestamps  | false     |
 | no-rows-affected    | false     |
 | no-driver-templates | false     |
+| ignore-fields       | []        |
```

Comment:
The naming of this is ambiguous. It must somehow refer to struct tags.

Reply:
I can rename it to `tag-ignore` and put a better description of usage.